### PR TITLE
update changelog for edge-19.10.5

### PR DESCRIPTION
**Pull request by hawkw**
## edge-19.10.5

This edge release adds support for integrating Linkerd's public-key
infrastructure with an external certificate issuer such as [`cert-manager`],
adds distributed tracing support to the Linkerd control plane, and adds
protection against DNS rebinding attacks to the web dashboard. In addition, it
includes several improvements to the Linkerd CLI.

* CLI
  * Added a new `--identity-external-issuer` flag to `linkerd install` that
    configures Linkerd to use certificates issued by an external certificate
    issuer (such as `cert-manager`)
  * Added support for injecting a namespace to `linkerd inject` (thanks
    @mayankshah1607!)
  * Added checks to `linkerd check --preinstall` ensuring Kubernetes Secrets
    can be created and accessed
  * Fixed `linkerd tap` sometimes displaying incorrect pod names for unmeshed
    IPs that match multiple running pods
* Controller
  * Added support for using trust anchors from an external certificate issuer
    (such as `cert-mananger`) to the `linkerd-identity` service
* Web UI
  * Added `Host:` header validation to the `linkerd-web` service, to protect
    against DNS rebinding attacks
* Internal
  * Added new `--trace-collector` and `--trace-collector-svc-account` flags to
    `linkerd inject` that configures the OpenCensus trace collector used by
    proxies in the injected workload (thanks @Pothulapati!)
  * Added a new `--control-plane-tracing` flag to `linkerd install` that enables
    distributed tracing in the control plane (thanks @Pothulapati!)
  * Added distributed tracing support to the control plane (thanks
    @Pothulapati!)

Also, thanks to @joakimr-axis for several fixes and improvements to internal
build scripts!

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,10 +1,47 @@
...
+  * Added new `--trace-collector` and `--trace-collector-svc-account` flags to
+    `linkerd inject` that configures the OpenCensus trace collector used by
+    proxies in the injected workload (thanks @ Pothulapati!)
+  * Added a new `--control-plane-tracing` flag to `linkerd install` that enables
+    distributed tracing in the control plane (thanks @Pothulapati!)
+  * Added support for injecting a namespace to `linkerd inject` (thanks
+    @mayankshah1607!)
+  * Added checks to `linkerd check --preinstall` ensuring Kubernetes Secrets
+    can be created and accessed
+  * Fixed `linkerd tap` sometimes not displaying incorrect pod names for
```
Comment (scottcarol) [inline CHANGES.md]: should be "sometimes displaying incorrect..."

File: CHANGES.md [hunk 2]
```diff
@@ -1,10 +1,47 @@
+## edge-19.10.5
+
+This edge release adds support for integrating Linkerd's PKI with an external
```
Comment (scottcarol) [inline CHANGES.md]: will everyone know what PKI means? is it worth spelling it out?

File: CHANGES.md [hunk 3]
```diff
@@ -1,10 +1,47 @@
...
+    `linkerd inject` that configures the OpenCensus trace collector used by
+    proxies in the injected workload (thanks @ Pothulapati!)
+  * Added a new `--control-plane-tracing` flag to `linkerd install` that enables
+    distributed tracing in the control plane (thanks @Pothulapati!)
+  * Added support for injecting a namespace to `linkerd inject` (thanks
+    @mayankshah1607!)
+  * Added checks to `linkerd check --preinstall` ensuring Kubernetes Secrets
+    can be created and accessed
+  * Fixed `linkerd tap` sometimes not displaying incorrect pod names for
+    unmeshed IPs that match multiple running pods
```
Comment (scottcarol) [inline CHANGES.md]: i might delete the words from "that match multiple..." to the end of the sentence, that level of detail seems unnecessary

Comment (hawkw) [reply CHANGES.md]: I thought it was worth explaining the cases in which the bug occurs, but I'm happy to remove it!

File: CHANGES.md [hunk 4]
```diff
@@ -1,10 +1,47 @@
+## edge-19.10.5
+
+This edge release adds support for integrating Linkerd's PKI with an external
+certificate issuer such as [`cert-manager`], adds distributed tracing support to
```
Comment (ihcsim) [inline CHANGES.md]: @grampelberg Any thoughts on whether we want to call out control plane distributed tracing here? Currently, the CLI flag is hidden, for dev/debugging purposes only. I wonder if we should either leave it out or label it as _internal_.

File: CHANGES.md [hunk 5]
```diff
@@ -1,10 +1,47 @@
...
+CLI.
+
+* CLI
+  * Added a new `--identity-external-issuer` flag to `linkerd install` that
+    configures Linkerd to use certificates issued by an external certificate
+    issuer (such as `cert-manager`)
+  * Added new `--trace-collector` and `--trace-collector-svc-account` flags to
+    `linkerd inject` that configures the OpenCensus trace collector used by
+    proxies in the injected workload (thanks @ Pothulapati!)
+  * Added a new `--control-plane-tracing` flag to `linkerd install` that enables
```
Comment (ihcsim) [inline CHANGES.md]: See comment above regarding control plane tracing.

File: CHANGES.md [hunk 6]
```diff
@@ -1,10 +1,47 @@
+## edge-19.10.5
+
+This edge release adds support for integrating Linkerd's PKI with an external
+certificate issuer such as [`cert-manager`], adds distributed tracing support to
+the Linkerd control plane, and adds protection against DNS rebinding attacks to
+the web dashboard. In addition, it includes several improvements to the Linkerd
+CLI.
+
+* CLI
+  * Added a new `--identity-external-issuer` flag to `linkerd install` that
```
Comment (ihcsim) [inline CHANGES.md]: Sorry, no doc yet. Maybe add a note referring users to https://github.com/linkerd/linkerd2/pull/3600#issue-329672028 for example usage?

File: CHANGES.md (hunk 4 above)
Comment (grampelberg) [reply CHANGES.md]: My vote is internal, we should tell folks you can do it ... just not encourage it.

Comment (hawkw) [reply CHANGES.md]: @grampelberg in that case, should it be removed from the release summary?

File: CHANGES.md (hunk 2 above)
Comment (hawkw) [reply CHANGES.md]: I think people reading the Linkerd changelogs will probably be familiar with this acronym, but I'm happy to expand it.

**Pull request merged**